### PR TITLE
instantview 3.19R0002 (new cask)

### PR DESCRIPTION
**Pull request by kiblik**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --cask --new <cask>` worked successfully.
- [x] `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

---

### CODE REVIEW

File: Casks/i/instantview.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  url "https://www.siliconmotion.com/downloads/macOS_InstantView_#{version}.dmg"
+  name "InstantView"
+  desc "SiliconMotion: SM76x Driver with UI"
+  homepage "https://www.siliconmotion.com/"
+
+  livecheck do
+    url "https://www.siliconmotion.com/downloads/index.html"
+    regex(/macOS_InstantView_(V\d\..+).dmg/i)
+  end
```
Comment (razvanazamfirei) [inline Casks/i/instantview.rb:13]:
```suggestion
  livecheck do
    url "https://www.siliconmotion.com/downloads/index.html"
    regex(/macOS[._-]instantview[._-]v(\d+(?:\.\d+)+)(\w\d+)\.dmg/i)
    strategy :page_match do |page, regex|
      page.scan(regex).map { |match| "#{match[0]},#{match[1]}" }
    end
  end
```

File: Casks/i/instantview.rb [hunk 2]
```diff
@@ -0,0 +1,26 @@
+cask "instantview" do
+  version "V3.19R0002"
```
Comment (razvanazamfirei) [inline Casks/i/instantview.rb:2]:
```suggestion
  version "3.19,R0002"
```

File: Casks/i/instantview.rb [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  sha256 "b9c258ccca92281a0c942139f7e7053a655015c1221245c4a83b119e65cf0080"
+
+  url "https://www.siliconmotion.com/downloads/macOS_InstantView_#{version}.dmg"
```
Comment (razvanazamfirei) [inline Casks/i/instantview.rb:5]:
```suggestion
  url "https://www.siliconmotion.com/downloads/macOS_InstantView_V#{version.csv.first}#{version.csv.second}.dmg"
```

File: Casks/i/instantview.rb [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  name "InstantView"
+  desc "SiliconMotion: SM76x Driver with UI"
```
Comment (razvanazamfirei) [inline Casks/i/instantview.rb:7]: Make sure the description matches our [guidelines](https://docs.brew.sh/Cask-Cookbook#stanza-desc).

**Review by razvanazamfirei — CHANGES_REQUESTED** [review]
Hi @kiblik, 

Thank you for opening this! Some suggestions below.

File: Casks/i/instantview.rb (hunk 4 above)
Comment (kiblik) [reply Casks/i/instantview.rb]: Is this better? https://github.com/Homebrew/homebrew-cask/pull/168882/commits/75b3e98fbc621211c13dbee926922089dca8fc33

**Review by razvanazamfirei — APPROVED** [review]

**Comment by razvanazamfirei** [comment]
Amazing, thank you so much @kiblik!

**Pull request merged**